pkg/minikube/bootstrapper/bsutil/extraconfig.go

```diff
@@ -56,12 +56,12 @@ var componentToKubeadmConfigKey = map[string]string{
 	Kubelet: "",
 }
 
-// KubeadmExtraArgsWhitelist is a whitelist of supported kubeadm params that can be supplied to kubeadm through
+// KubeadmExtraArgsAllow is a list of supported kubeadm params that can be supplied to kubeadm through
```

Comment:
Some suggestions if you're open to them:

- `KubeadmExtraArgAllowlist` (it's a bit more specific that this is replacing the term "whitelist", which could be helpful)
- `KubeadmExtraArgsAllowed` (grammatically/semantically correct. `ExtraArgsAllow` to me implies a boolean, where as `ExtraArgsAllowed` implies a list of some sort.)

💖